pkg/restapi/issuer/operation/operations_test.go

```diff
@@ -646,6 +686,31 @@ func TestCreateProfile(t *testing.T) {
 
 		require.Contains(t, resErr.ErrMessage, "create oidc client")
 	})
+	t.Run("create profile - failed to output descriptors configured for waci profiles", func(t *testing.T) {
```

Comment:
You probably mean "failed to configure output descriptors for WACI profiles" instead